Exercise_A/style.css

```diff
@@ -0,0 +1,29 @@
+#container {

```

Comment:
Good idea to limit the height and the width, you have used floats well for the green and blue squares, though your red square uses margins to get it into position.
My advice here is that you should attempt to clear the float and have an auto margin for the red style block.

In addition to this, I would remove height: 100px and width: 100px  from every style block, either target a class with these values or the div tag.